Sparky-core/src/sp/platform/opengl/GLContext.cpp

```diff
@@ -42,4 +43,9 @@ namespace sp { namespace graphics { namespace API {
 		SwapBuffers(hDc);
 	}
 
+	void GLContext::SetVsync(const uint& syncInterval)
+	{
+		wglSwapIntervalEXT(syncInterval);
```

Comment:
I *think* the GL backend is intended to work on Windows, Mac, and Linux. This would only work on Windows.